src/composables/painter/usePainter.test.ts

```
@@ -0,0 +1,463 @@
+        setup() {
+          usePainter('test-node', {
+            canvasEl,
+            cursorEl,
+            modelValue
+          })
+          return {}
+        },
+        render() {
+          return null
+        }
+      })
```

ESLint

[error] There is more than one component in this file.

(vue/one-component-per-file)